src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -268,16 +268,20 @@ public void NetStandardReferenceTest ()
 using System.Linq;
 using System.Text;
 using Newtonsoft.Json;
+using System.IO.Packaging;
 
 using Xamarin.Forms;
 
 namespace XamFormsSample
 {
     public partial class App : Application
     {
+        Package package;
+
         public App()
         {
             JsonConvert.DeserializeObject<string>(""test"");
+            var package = Package.Open ("""");
```

Comment:
Wait, what? What does `""""` *mean*? `csharp` doesn't like that:

```bash
$ csharp
csharp> string s = ""test"";
(1,14): error CS1525: Unexpected symbol `test'
```

I don't understand how this and the previous line even work. :-/

Reply:
This code block is part of a string `@""` block [1]. To the `""""` are escaped double quotes. We need a variable to use `Package` otherwise the linker will strip the assembly from the final apk. This way we can make sure the required `System.IO.Packaging` assembly ends up in the .apk (for the test).

[1] https://github.com/dellis1972/xamarin-android/blob/6663746918fc2cd255c913d45092c9d7b1ec76d8/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs#L265

Reply:
> We need a variable to use `Package` otherwise the linker will strip the assembly from the final apk.

I still don't understand. We need a call to `Package.Open()` to preserve the type; that makes sense. That doesn't answer why the `App.package` field exists, nor does it answer why the `App.package` field *is ignored*. `App.package` will *always be `null`*, because the `var package` declaration within the `App` constructor *hides* the `App.package` field.

Reply:
ah right , sorry I get ya now lol